### PR TITLE
Update year in example copyright blurb

### PR DESCRIPTION
**Pull request by jakemac53**
Fixes https://github.com/dart-lang/test/issues/1498

I would remove the sentence about the AUTHORS file but I don't want to touch the substance of the blurb even though its obviously not providing value right now.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,17 +37,15 @@ new tests, use the existing tests as a guideline for what they should look like.
 Before you send your pull request, make sure all the tests pass! To run all the
 tests, you'll need:
 
-* [PhantomJS][] version 2.0.0 or higher.
+* [NodeJS]: https://nodejs.org/
 
-[PhantomJS]: http://phantomjs.org/
-
-Once you have these, just run `pub run test`.
+Once you have this, just run `pub run test`.
```
Comment (natebosch) [inline CONTRIBUTING.md:42]: This isn't true anymore.

We could replace it with `mono_repo presubmit` perhaps...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -37,17 +37,15 @@ new tests, use the existing tests as a guideline for what they should look like.
 Before you send your pull request, make sure all the tests pass! To run all the
 tests, you'll need:
 
-* [PhantomJS][] version 2.0.0 or higher.
+* [NodeJS](https://nodejs.org/])
```
Comment (natebosch) [inline CONTRIBUTING.md:40]:
```suggestion
* [NodeJS](https://nodejs.org/)
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (jakemac53) [reply CONTRIBUTING.md:42]: lol true, I could remove the entire section around testing... (the expectation I think is more to rely on CI than locally run)

Comment (natebosch) [reply CONTRIBUTING.md:42]: Removing the section would probably be fine.

Using CI is a more accurate reflection of what we do in practice.

Comment (jakemac53) [reply CONTRIBUTING.md]: I updated it to reference mono_repo as a tool, but not a requirement

**Review by natebosch — APPROVED** [review]

**Pull request merged**